src/cf-base.less

```diff
@@ -536,26 +506,18 @@ table {
 
 th,
 td {
-    padding: unit(12px / @base-font-size-px, em) unit(15px / @base-font-size-px, em);
-    background: @td-bg;
+    padding: unit( 10px / @base-font-size-px, em);
```

Comment:
Trivial spacing issue: remove space after opening parenthesis. (Won't hold up the PR, though.)